index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>Editor Staff Hub</title>
+  <meta charset="utf-8">
+  <meta name="author" content="Editor Staff Hub j4502">
+  <meta name="description" content="Editor Staff Hub">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+ </head>
+
+<body>
+<img src="assets/images/logo.png" alt="Columbia Missourian Logo">
+<img src="assets/images/weather.png" alt="Weather"/>
+
+    <header>
+      <h3>Missourian Editor Staff Hub</h3>
+
+      <nav>
+      <ul>
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">Bios</a></li>
+        <li><a href="#">Join the Conversation</a></li>
+        <li><a href="#">Subscribe</a></li>
+        <li><a href="#">Advertise With Us</a></li>
+        <li><a href="#">History</a></li>
+        <li><a href="#">From the Newsroom</a></li>
+      </ul>
+
+    </nav>
+    </header>
+<div>
+  <h1> Contact Us </h1>
+</div>
+    <div>
```

Comment:
So this really needs to be much higher on the page, like the first thing that people see when they get here. It would also be nice to add in the email addresses for these places.